app/component/ItineraryPage.js

```diff
@@ -259,8 +256,6 @@ class ItineraryPage extends React.Component {
           result.bikeAndPublicPlan?.itineraries,
         );
 
-        // show 6 bike + transit itineraries, preferably 3 of both kind.
-        // If there is not enough of a kind, take more from the other kind
```

Comment:
Why is this removed now?

Reply:
Strange. Must be some git merge thing. Restored now.